src/test/java/org/apache/accumulo/proxy/its/SimpleProxyBase.java

```diff
@@ -162,13 +163,12 @@ protected Duration defaultTimeout() {
   private TestProxyClient proxyClient;
   private org.apache.accumulo.proxy.thrift.AccumuloProxy.Client client;
 
-  private static final Map<String,String> properties = new HashMap<>();
   private static String hostname;
   private static String proxyPrimary;
   private static String clientPrincipal;
   private static File clientKeytab;
 
-  private ByteBuffer creds = null;
+  private static String creds;
```

Comment:
`creds` is another one whose name is overused and needs more clarity. In this case, it's storing the shared secret for the proxy.